db/migrations/20210713211341_add_id_user_document.sql

```diff
@@ -0,0 +1,10 @@
+-- migrate:up
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp" WITH SCHEMA public;
```

Comment:
You shouldn't need to create this again, everything from our previous migration will persist.